sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -1307,6 +1308,69 @@ class AdaptiveQueryExecSuite
     }
   }
 
+  test("SPARK-35455: Enhance EliminateUnnecessaryJoin - single join") {
```

Comment:
let's update the test name and PR title: `Unify empty relation optimization between normal and AQE optimizer`

Reply:
Updated it and also updated the PR title.